components/movementsensor/gpsrtk/gpsrtk.go

```diff
@@ -21,18 +21,6 @@ import (
 	"go.viam.com/rdk/spatialmath"
 )
 
-// NtripConfig is used for converting attributes for a correction source.
```

Comment:
Since this package no longer has a model, let's rename it to `rtkutils` or something to express its function better.

Reply:
Might break @oliviamiller's module so let's make it a lasting name change and coordinate.

Reply:
great.